vars/deps_SUSE.yml

```diff
@@ -0,0 +1,17 @@
+---
+# ©Copyright 2015-2016 Hewlett Packard Enterprise Development Company, LP
+
+
+monasca_os_dependencies:
+  - python-devel
+  - python-pip
+  - python-virtualenv
+  - python-PyYAML
+  - libxml2-devel
+  - libxslt-devel
+  - libffi-devel
+  - openssl-devel
+  - gcc
+  - lshw
+
+...
```

Comment:
Nit: remove these?

Reply:
It's true I made no effort to test SUSE package installs, but should we address the question of removing SUSE support in a separate PR?

Reply:
What's with the `...` though?